frontend/public/components/resource-quota.jsx

```diff
@@ -13,6 +13,100 @@ const menuActions = [...common];
 
 const quotaKind = quota => quota.metadata.namespace ? referenceForModel(ResourceQuotaModel) : referenceForModel(ClusterResourceQuotaModel);
 
+const humanizeQuotaResource = (resourceType, useTitleCase) => {
+  if (!resourceType) {
+    return resourceType;
+  }
+
+  const nameTitleCaseFormatMap = {
+    'configmaps': 'Config Maps',
+    'cpu': 'CPU (Request)',
+    'limits.cpu': 'CPU (Limit)',
+    'limits.memory': 'Memory (Limit)',
+    'memory': 'Memory (Request)',
+    'openshift.io/imagesize': 'Image Size',
+    'openshift.io/imagestreamsize': 'Image Stream Size',
+    'openshift.io/projectimagessize': 'Project Image Size',
+    'persistentvolumeclaims': 'Persistent Volume Claims',
+    'requests.storage': 'Storage (Request)',
+    'pods': 'Pods',
+    'replicationcontrollers': 'Replication Controllers',
+    'requests.cpu': 'CPU (Request)',
+    'requests.memory': 'Memory (Request)',
+    'resourcequotas': 'Resource Quotas',
+    'secrets': 'Secrets',
+    'services': 'Services',
+    'services.loadbalancers': 'Service Load Balancers',
+    'services.nodeports': 'Service Node Ports',
+  };
+
+  const nameFormatMap = {
```

Comment:
Better to have one map that has both lowercase and title case values. That way we can make sure the maps have all the same keys.

```js
const quotaResourceLabels = Object.freeze({
  configmaps: { label: 'config maps', titleCase: 'Config Maps' },
  // ...
});

const humanizeQuotaResource = (resourceType, useTitleCase) =>
  _.get(quotaResourceLabels, [resourceType, useTitleCase ? 'titleCase' : 'label'], resourceType);
```

Reply:
fixed